File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/rest-codestart/java/src/main/java/org/acme/{resource.class-name}.tpl.qute.java
package org.acme;

import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;
import jakarta.ws.rs.Produces;
import jakarta.ws.rs.core.MediaType;

@Path("{resource.path}")
public class {resource.class-name} {

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    public String hello() {
        return "{resource.response}";
    }
}

File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/rest-codestart/kotlin/src/main/kotlin/org/acme/{resource.class-name}.tpl.qute.kt
package org.acme

import jakarta.ws.rs.GET
import jakarta.ws.rs.Path
import jakarta.ws.rs.Produces
import jakarta.ws.rs.core.MediaType

@Path("{resource.path}")
class {resource.class-name} {

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    fun hello() = "{resource.response}"
}